### PR TITLE
home work 1_basic HTML and CSS

### PR DESCRIPTION
**Pull request by Lytvynenko18**

### CODE REVIEW

File: homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                <p class="about_me_text"> Hi! My name is Roman Lytvynenko and I'm a Junior Frontend Developer. I am already familiar with main Web Technologies like HTML, CSS, JavaScript and Git version control system.
+                </p>
+                <p
+                    class="about_me_text">This page was developed during the course <a class="about_me_links" href="https://www.mastersacademy.education/frontend-for-beginners-it"> 'Frontend for beginners' from Masters Academy in 2023. </a>
+                </p>
+                <p
+                    class="about_me_text"> This is a social project from MOCG company where I got an opportunity to work with Frontend mentors and to create my own small project for the portfolio.
+                </p>
+                <p class="about_me_text"> You can contact me via
+                    <a class="about_me_links" href="https://google.com">Google</a> and/or check out my GitHub.
```
Comment (vadym-zinchenko-moc) [inline homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/index.html:24]: pls add github link, and instead Google your social media pls

File: homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/style.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
+.about_me_article {
```
Comment (vadym-zinchenko-moc) [inline homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/style.css:1]: read pls about BEM methodology and change names of classes

**Review by vadym-zinchenko-moc — CHANGES_REQUESTED** [review]

File: homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/style.css (hunk 2 above)
Comment (Lytvynenko18) [reply homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/style.css:1]: done

File: homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/index.html (hunk 1 above)
Comment (Lytvynenko18) [reply homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/index.html]: done

Comment (Lytvynenko18) [reply homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/index.html]: Should I close this pull request and create new one?

Comment (vadym-zinchenko-moc) [reply homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/index.html]: Good job. Approved.

**Review by vadym-zinchenko-moc — APPROVED** [review]

**Review by vadym-zinchenko-moc — APPROVED** [review]

**Pull request merged**